spec/schema_tools/hash_spec.rb

```diff
@@ -217,6 +225,27 @@ class NestedObjectNoProperties
       hash['person']['first_name'].should == 'Pit'
     end
 
+    it 'has nested anyOf type object ' do
```

Comment:
Im not sure I follow, it just loads a different schema, one includes an anyOf node, the other includes an allOf node